builtin/aws/ecs/platform.go

```diff
@@ -1567,6 +1572,9 @@ type Config struct {
 	// Subnets to place the service into. Defaults to the subnets in the default VPC.
 	Subnets []string `hcl:"subnets,optional"`
 
+	// Security Groups ARN to define existing security groups for ecs.
```

Comment:
Does the ARN work here? When I tested this I had to use the Security Group ID.

Reply:
Sorry, I'm confused about the concepts. Fixed